tla/consensus/ccfraft.tla

```diff
@@ -981,20 +981,23 @@ AppendEntriesAlreadyDone(i, j, index, m) ==
                 IF membershipState = RetirementSigned /\ commitIndex' > RetirementIndex(i) 
                 THEN RetirementCompleted 
                 ELSE @]
-    /\ isNewFollower' = [isNewFollower EXCEPT ![i] = FALSE]
     /\ Reply([type           |-> AppendEntriesResponse,
               term           |-> currentTerm[i],
               success        |-> TRUE,
               lastLogIndex   |-> m.prevLogIndex + Len(m.entries),
               source         |-> i,
               dest           |-> j],
               m)
-    /\ UNCHANGED <<removedFromConfiguration, currentTerm, leadershipState, votedFor, log, candidateVars, leaderVars>>
+    /\ UNCHANGED <<removedFromConfiguration, currentTerm, leadershipState, votedFor, isNewFollower, log, candidateVars, leaderVars>>
 
 \* Follower i receives an AppendEntries request m where it needs to roll back first
 \* This action rolls back the log and leaves m in messages for further processing
 ConflictAppendEntriesRequest(i, index, m) ==
     /\ Len(log[i]) >= index
+    /\ m.entries /= << >>
+    /\ \E idx \in 1..Len(m.entries) :
+        /\ (index + (idx - 1)) \in DOMAIN log[i]
+        /\ log[i][index + (idx - 1)].term # m.entries[idx].term
```

Comment:
After removing the `isNewFollower` reset from elsewhere in the spec, this enablement condition is required to pass simulation. I believe that without this, the spec says that _any_ AppendEntries can be treated a though it conflicts, and cause a rollback. This was never an issue previously, because simulation includes no message loss:
- The first AE in each term is a heartbeat
- Taking the conflict path on a heartbeat is a safe nop - we don't actually truncate any log
- After processing the AE by any path, we reset `isNewFollower[i]`
- Any future AEs in this term cannot take this conflict path, due to the `isNewFollower[i] = TRUE` enablement condition

But now we need to match the implementation's "one rollback message processed per-term" behaviour, for trace validation, so I think we need this enablement condition for simulation.